### DISCUSSION TITLE
Notifications for for taking a brake, going home, turning of the pc and spent time with loved ones

### DISCUSSION THREAD

**Discussion by EN20M** [ActivityWatch/activitywatch#661] [Ideas]
I wanted to start an open source project and needed some help and someone on reddit recommended having a look at this project. I think my idea of a tool that is more trying to help you break focus rather than track what you are doing is kind of related but I do see that it has a different base idea. If you think this fits in here I would be more than happy to join in an and contibute.

I have a whacky prototype/proof of concept but it really lacks GUI and thats where i definetly would need help.
[WorkTimeTracker Prototyp](https://gitlab.com/EN20MARS/work-time-tracker)

**Comment by ErikBjare** [comment]
That's a cool idea!

I've thought about other uses for notifications, in particular the ability to get notifications depending on some goal, like having spent some number of hours of x (games, work, etc) on that day/week.

It would be cool if we could build a general solution that lets you do both, depending on how its configured.

**Reply by EN20M** [reply]
Thats awsome mabe we can have a chat some time! I created a discord server for my project [everyone is welcome to join](https://discord.gg/gDfEK4ut) , I'm usually around in the evening (GMT+1).

Edit: As I just noticed you have a discord as well so I will try to catch you there.
